Sources/VLCUI/VLCVideoPlayer/Proxy.swift

```diff
@@ -152,5 +152,42 @@ public extension VLCVideoPlayer {
         public func playNewMedia(_ newConfiguration: Configuration) {
             videoPlayerView?.setupVLCMediaPlayer(with: newConfiguration)
         }
+
+        /// Starts the recording process
+        /// - Parameter directoryPath: The directory path where the recording will be saved
+        /// - Returns: Indicates whether the recording started successfully
+        public func startRecording(directoryPath: String) -> Bool {
+            guard let mediaPlayer else { return false }
+            guard mediaPlayer.isPlaying else { return false }
```

Comment:
It's fine to start a recording if media hasn't started playing or is paused, it will just record the portion that gets played.